### ISSUE TITLE
Add Linux arm64 support

### ISSUE THREAD

**Issue by dromer** [plugdata-team/plugdata-heavy-toolchain#20]
We are starting to see people wanting to use plugdata and the Heavy toolchain on arm64, so we may want to figure out how to add this architecture.

**Comment by JoshuaACNewman** [comment]
I've got academic computer science contacts over here. Do you know where the gaps are?

**Comment by dromer** [comment]
We need to replace/reimplement https://github.com/theopolis/build-anywhere

**Comment by JoshuaACNewman** [comment]
Are you suggesting that I shouldn't ask interested parties?

**Comment by dromer** [comment]
No, but said qualifications should not depend on where they work :P
